assets/customize-settings.js

```diff
@@ -1,43 +1,52 @@
-/*
- * Custom JS to modify the Altis CMS module Add New Site page
- */
-const siteSubdomain = document.getElementById( 'site-subdomain' ),
-	siteSubdirectory = document.getElementById( 'site-subdirectory' ),
-	siteCustomDomain = document.getElementById( 'site-custom-domain' ),
-	domainTypeRadioBtns = document.getElementsByName( 'domain-type' ),
-	siteAddress = document.getElementById( 'site-address' ),
-	siteAddressDesc = document.getElementById( 'site-address-desc' ),
-	networkAddress = window.location.hostname;
-let domainTextEl = document.createElement( 'span' );
-domainTextEl.className = 'site-address-domain-text';
+( function () {
+	/*
+	* Custom JS to modify the Altis CMS module Add New Site page
+	*/
+	const siteSubdomain = document.getElementById( 'site-subdomain' );
+	const siteSubdirectory = document.getElementById( 'site-subdirectory' );
+	const siteCustomDomain = document.getElementById( 'site-custom-domain' );
+	const siteAddress = document.getElementById( 'site-address' );
+	const siteAddressDesc = document.getElementById( 'site-address-desc' );
+	const networkAddress = window.location.hostname;
```

Comment:
Can the network admin be accessed via subsite or always on the primary hostname?

Reply:
Yes, but I think this matches what WordPress already does? In any case, I didn't change this code, just refactored.